upload_s3.sh

```diff
@@ -29,7 +29,7 @@ for file in $(find "$GITHUB_SHA" -type f); do
 
```

Comment:
I was going to suggest also changing the `exit 1` at the top, but realized that if that happened, it should probably fail noisily...